packages/google_maps_flutter/google_maps_flutter_android/example/integration_test/google_maps_tests.dart

```diff
@@ -995,7 +995,7 @@ void googleMapsTests() {
   },
       // TODO(cyanglaz): un-skip the test when we can test this on CI with API key enabled.
       // https://github.com/flutter/flutter/issues/57057
-      skip: Platform.isAndroid);
+      skip: true);
```

Comment:
Did you mean to skip this for all platforms? if so should it be deleted instead?

Reply:
No, this was committed by accident. Not related to this PR.

Reply:
This is a legacy of this file starting its life as a copy of the integration tests in `google_maps_flutter`. In the context of `google_maps_flutter_android`, `Platform.isAndroid` is just a confusing way of writing `true`, so this change was good.

> if so should it be deleted instead?

I would prefer we keep it, because of the TODO above. We are starting to see features that do not work without an API key, so we will need to fix the CI limitation at some point. (I was working on it with infra folks a while ago, but that got derailed.)

Reply:
This is true; but already reverted this change before I saw @stuartmorgan:n comment about this, as this was chore change, not related to ground overlay at all. I can commit this back, if it is ok to have unrelated changes in the PR as well.

Reply:
It's fine either way; if you've already reverted it don't worry about adding it back unless you feel like it.